sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -13,14 +13,18 @@
   `_OTELRESOURCE_` metrics.
   ([#41761](https://github.com/Azure/azure-sdk-for-net/pull/41761))
 
+* Fixed an issue where additional tags provider with `activity.RecordException`
```

Comment:
```suggestion
* Fixed an issue where additional tags provided with `activity.RecordException`
```

Reply:
It is not correct to say provided with activity.RecordException, as it is just a helper/extension method. More apt is to say, Tags provided as `ActivityEvent`, following exception semantic convention, is now exported as additional kvp. This will cover folks who directly add ActivityEvent, without going through RecordException API.